### PR TITLE
Use cases

### PR DESCRIPTION
**Pull request by mettelilienthal**

### CODE REVIEW

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [inline content/terms_and_definitions/terms_and_definitions.adoc:1]: Please remove the changes here, as they are independent from the use cases.

**Comment by ClemensLinnhoff** [comment]
@mettelilienthal please correct your DCO signoff

File: content/use_cases/use_cases.adoc [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+|*Influencing Standards*    |
+|*Influencing Technologies* |
+|*Used 3D Model Formats*    |
+|*Used Material Formats*    |
+|*Companies*                |
+|*Relevant User*                 |
+|===
+
+
+////
```
Comment (ClemensLinnhoff) [inline content/use_cases/use_cases.adoc:210]: The commented use-cases can be removed. The topics are already covered by the other use-cases.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: @mettelilienthal This is still open. The terms and definitions file can be removed from this PR, right?

File: content/use_cases/use_cases.adoc (hunk 1 above)
Comment (ClemensLinnhoff) [reply content/use_cases/use_cases.adoc:210]: The commented use-cases still show up in the changes.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (mettelilienthal) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: We're tidying it up right now, yes

File: content/use_cases/use_cases.adoc (hunk 1 above)
Comment (mettelilienthal) [reply content/use_cases/use_cases.adoc]: We've pushed all changes; you should be able to see the correct (current) version now.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (FKlopfer) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: terms and definitions file is removed from this PR

File: content/use_cases/use_cases.adoc (hunk 1 above)
Comment (ClemensLinnhoff) [reply content/use_cases/use_cases.adoc]: Now the file is removed, so it is still changed by this PR.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: Now the file is removed, so it is still changed by this PR.

File: content/use_cases/use_cases.adoc (hunk 1 above)
Comment (ClemensLinnhoff) [reply content/use_cases/use_cases.adoc]: Looks good, thanks.

File: content/terms_and_definitions/terms_and_definitions.adoc
Comment (FKlopfer) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: Yes, sorry for the misunderstanding. Reverted the changes on the file. It is not visible in the PR anymore.

Comment (ClemensLinnhoff) [reply content/terms_and_definitions/terms_and_definitions.adoc:1]: Thanks.

**Review by ClemensLinnhoff — APPROVED** [review]

**Comment by ClemensLinnhoff** [comment]
This PR is ready to be review by the CCB.

**Comment by ClemensLinnhoff** [comment]
CCB decision 26.06.24: Merge as is

**Pull request merged**